bin/runner.js

```diff
@@ -19,14 +19,16 @@ try {
 
         throw err;
       })
+      .then(() => runner.startTunnel())
```

Comment:
Not sure if we should start the tunnel here, we only need the tunnel while using browserstack transport. Try starting the tunnel from `createSession` of *browserstack.js*

Reply:
@beatfactor what do you think?

Reply:
In case of parallel, `createSession` runs for each session. Local will try to spawn binary multiple times resulting `local already running error`.

Reply:
Good point. But even with the current implementation, the binary will be spawned multiple times. This is because for every job we start a new nightwatch process. What can be done to go around that is to pass in the `localIdentifier` as a parameter in `concurrency.js`